compiler/damlc/tests/src/DA/Test/DamlcIntegration.hs

```diff
@@ -332,40 +332,14 @@ parseRange :: String -> Range
 parseRange s =
   case traverse readMaybe (wordsBy (`elem` (":-" :: String)) s) of
     Just [rowStart, colStart, rowEnd, colEnd] ->
-      -- When specifying ranges:
-      --  * lines are 1-based
-      --  * columns are 0-based
-      --  * the column span is open, that is, the end column is "one
-      --    past the end" of the span.
-      --
-      -- Positions in these ranges are matched against LSP coordinates
-      -- which are 0-based in both line and column and also open. Error
-      -- messages are formatted by GHC and are 1-based in line, 0-based
-      -- in column and closed (so add one to column start, one to
```

Comment:
Does the comment about GHC still apply? i.e., will we now get locations in error messages that are 0 based column-wise while the location of the diagnostics containing that error message is 1 based? At the very least it seems worth preserving the comment that documents this behavior.

Reply:
This comment was never true. GHC's ranges were always 1-based and hence match the way we pretty-print diagnostics now:
https://github.com/digital-asset/daml-ghcide/blob/8e4f52892d88d23259547b03bb096b46f28c0afc/src/Development/IDE/GHC/Error.hs#L70

I think all the confusion came from the bug in the pretty-printer I've fixed yesterday. Since we don't use any `SrcSpan`s in this test but only `Range`s, I think the comment makes no sense in this place anymore. In view of https://github.com/digital-asset/daml/pull/7708, there should also be no more confusion for our devs who want to add or fix tests, since we don't print anything 0-based anymore.

Reply:
makes sense, thanks for the explanation